Dockerfile

```diff
@@ -51,6 +51,9 @@ RUN ./build/install_gems.sh
 # Add the application code
 ADD . /data
 
+RUN cp /data/app/assets/html/404.html.haml public/404.html
+RUN cp /data/app/assets/html/500.html.haml public/500.html
+
 RUN if [ "${RAILS_ENV}" = "production" ] || [ "${RAILS_ENV}" = "staging" ]; then \
   RAILS_ENV=${RAILS_ENV} SECRET_KEY_BASE=${SECRET_KEY_BASE} bundle exec rails assets:precompile; \
```

Comment:
Here's where assets are compiled for production. Add a `cp` or `rsync` here to move /app/assets/\*.html to /public/\*.html
Maybe add an else for copying .haml files when not on prod/staging, instead of above.